src/Features/CSharp/Portable/UseIsNullCheck/CSharpUseIsNullCheckForEqualityOperatorDiagnosticAnalyzer.cs

```diff
@@ -9,13 +9,10 @@
 namespace Microsoft.CodeAnalysis.CSharp.UseIsNullCheck
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
-    internal class CSharpUseIsNullCheckForCastAndEqualityOperatorDiagnosticAnalyzer
+    internal class CSharpUseIsNullCheckForEqualityOperatorDiagnosticAnalyzer
```

Comment:
we should not cahnge teh behavior of this analyzer.  The feature is documented as:

![image](https://user-images.githubusercontent.com/4564579/64479062-5685f600-d166-11e9-9ec5-fff877ab6499.png)

Reply:
basically, we're saying: instead of these idiomatic ways of asking for a reference-null check, use this idiomatic way to do a reference-null check.  i.e. the user *originally* decided that `a == null` was not what they wanted.  However, in this PR we're actually taking their positive decision to use `a == null` and telling them to not do that.